patch/kconfig-secure-boot-inclusions

```diff
@@ -11,7 +11,7 @@ CONFIG_MODULE_SIG_FORCE=y
 CONFIG_SYSTEM_TRUSTED_KEYS="debian/certs/debian-uefi-certs.pem"
 CONFIG_MODULE_SIG_HASH="sha512"
 CONFIG_MODULE_SIG_SHA512=y
-CONFIG_KEXEC_SIG_FORCE=y
+CONFIG_KEXEC_SIG=y
```

Comment:
Please make this a separate commit.

Reply:
I moved those changes into a separate PR: https://github.com/sonic-net/sonic-linux-kernel/pull/381